src/test/java/htsjdk/samtools/CRAMComplianceTest.java

```diff
@@ -224,6 +245,15 @@ public void testCRAMThroughBAMRoundTrip(final File originalCRAMFile, final File
         writeRecordsToFile(bamRecords, tempCRAMFile, referenceFile, samHeader);
         List<SAMRecord> roundTripCRAMRecords = getSAMRecordsFromFile(tempCRAMFile, referenceFile);
 
+        i = 0;
+        for (SAMRecord rec : roundTripCRAMRecords) {
+            if (!rec.equals(originalCRAMRecords.get(i))) {
+                System.out.println(originalCRAMRecords.get(i).getSAMString() + rec.getSAMString());
+                rec.equals(originalCRAMRecords.get(i));
```

Comment:
What's going on with this block, it seems like it just dumps stuff to stdout?  Is it meant to assert something?  Also, what's going on with this `rec.equals` that is ignored?